src/qz/printer/action/PrintHTML.java

```diff
@@ -43,16 +43,29 @@ public void parseData(JSONArray printData, PrintOptions options) throws JSONExce
 
                 PrintingUtilities.Format format = PrintingUtilities.Format.valueOf(data.optString("format", "FILE").toUpperCase());
 
-                double pageWidth = PrinterJob.getPrinterJob().getPageFormat(null).getWidth();
+                double pageZoom = pxlOpts.getDensity() / 72.0;
+                if (pageZoom <= 0) { pageZoom = 1; }
+
+                double pageWidth = 0;
+                double pageHeight = 0;
+
+                // web dimension use 96dpi (or equivalent dp/metric)
```

Comment:
Hmm... Perhaps I'm reading the wrong portion, but the part I see says `px` is relative to the viewing device... 

> px
Relative to the viewing device.
For screen display, typically one device pixel (dot) of the display.
For printers and very high resolution screens one CSS pixel implies multiple device pixels, so that the number of pixel per inch stays around 96.

I guess I'm still confused as to how this was discovered, what impact it would have had on previous builds vs. new builds, etc.